### PR TITLE
[KOGITO-115] - reading properties during process execution

### PR DESCRIPTION
**Pull request by ricardozanini**
This PR depends on: https://github.com/kiegroup/kogito-cloud/pull/28

### CODE REVIEW

File: onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java [hunk 1]
```diff
@@ -9,29 +9,33 @@
 import org.kie.kogito.cloud.workitems.ServiceInfo;
 
 public class DecisionTaskWorkItemHandler extends DiscoveredServiceWorkItemHandler {
-
-    private String namespace = System.getenv("NAMESPACE");
+    
+    private boolean localServicesAdded;
```
Comment (mswiderski) [inline onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java]: why do we need this flag?

File: onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java [hunk 2]
```diff
@@ -9,29 +9,33 @@
...
+            this.addServices("paymentDate", new ServiceInfo("http://localhost:8082/paymentDate", null));
+            this.localServicesAdded = true;
+        } 
     }
 
     @Override
     public void executeWorkItem(WorkItem workItem, WorkItemManager manager) {
-
-        Map<String, Object> results = discoverAndCall(workItem, namespace, "Decision", HttpMethods.POST);
+        this.addLocalServices();
```
Comment (mswiderski) [inline onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java]: why do we need to do it at execute work item and not in the constructor? this is just static configuration so should not cause any additional look ups...

Comment (ricardozanini) [reply onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java]: I don't like this approach either. The problem is that the map can't hold the items added to it in the constructor, that's why we have to do this check every time. It's a different problem than the `System.getenv` one.

File: onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java (hunk 1 above)
Comment (ricardozanini) [reply onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java]: To not add the items in every process execution

File: onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java (hunk 2 above)
Comment (mswiderski) [reply onboarding-example/onboarding/src/main/java/org/kie/kogito/examples/onboarding/DecisionTaskWorkItemHandler.java]: what's the reason for this?

**Review by mswiderski — APPROVED** [review]

**Pull request merged**